src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/hooks/useCreateDrawerForm.tsx

```diff
@@ -289,6 +289,12 @@ const useCreateDrawerForm = (
           template,
           ANNOTATIONS.displayName,
         );
+
+        ensurePath(tabsDataDraft, 'disks.disks.rootDiskRegistryCredentials');
```

Comment:
the path you ensure here doesn't match the path you assign below

Reply:
Fixed!